recipes/libffi/all/conandata.yml

```diff
@@ -1,17 +1,19 @@
 sources:
+  "3.4.6":
+    url: "https://github.com/libffi/libffi/releases/download/v3.4.6/libffi-3.4.6.tar.gz"
+    sha256: "b0dea9df23c863a7a50e825440f3ebffabd65df1497108e5d437747843895a4e"
   "3.4.4":
     url: "https://github.com/libffi/libffi/releases/download/v3.4.4/libffi-3.4.4.tar.gz"
     sha256: "d66c56ad259a82cf2a9dfc408b32bf5da52371500b84745f7fb8b645712df676"
-  "3.4.3":
-    url: "https://github.com/libffi/libffi/releases/download/v3.4.3/libffi-3.4.3.tar.gz"
-    sha256: "4416dd92b6ae8fcb5b10421e711c4d3cb31203d77521a77d85d0102311e6c3b8"
-  "3.4.2":
-    url: "https://github.com/libffi/libffi/releases/download/v3.4.2/libffi-3.4.2.tar.gz"
-    sha256: "540fb721619a6aba3bdeef7d940d8e9e0e6d2c193595bc243241b77ff9e93620"
   "3.3":
     url: "https://github.com/libffi/libffi/releases/download/v3.3/libffi-3.3.tar.gz"
     sha256: "72fba7922703ddfa7a028d513ac15a85c8d54c8d67f55fa5a4802885dc652056"
 patches:
+  "3.4.6":
+    - patch_file: "patches/0002-3.4.6-fix-libtool-path.patch"
```

Comment:
Please start adding `patch_description` and `patch_type`. Additionally `patch_source` if applicable

Reply:
These were from existing patches, but I can do some digging and figure out their purposes.